src/deleted.psm1

```diff
@@ -1,3 +1,9 @@
+function IsNumber($var)
```

Comment:
We have this same problem in several other areas. We should consider moving this to ps-utilities.psm and use it to fix other functions.